runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerBox.java

```diff
@@ -215,13 +218,24 @@ void rebuild()
 			priceTypeString = priceType == LootTrackerPriceType.HIGH_ALCHEMY ? "HA: " : "GE: ";
 		}
 
-		priceLabel.setText(priceTypeString + QuantityFormatter.quantityToStackSize(totalPrice) + " gp");
-		priceLabel.setToolTipText(QuantityFormatter.formatNumber(totalPrice) + " gp");
+		String averageString = kills > 1 ? QuantityFormatter.formatNumber(totalPrice / kills) + " gp" : "0";
+		String totalString = priceTypeString + QuantityFormatter.formatNumber(totalPrice) + " gp";
+		if (showAverage)
+		{
+			priceLabel.setText(averageString);
+			priceLabel.setToolTipText(totalString + " (total)");
+		}
+		else
+		{
+			priceLabel.setText(priceTypeString + QuantityFormatter.quantityToStackSize(totalPrice));
+			priceLabel.setToolTipText("<html>" + QuantityFormatter.formatNumber(totalPrice) + " gp<br>" + averageString + " (average)</html>");			
+		}		
```

Comment:
Why does this change add the average string to the tooltip even when the configuration option is disabled?

Reply:
@Nightfirecat  - I saw the setting as the displayed value, and the tooltip as a means to show the opposite.  2 reasons for doing that:
1. I assume it won't be the option that people will want displayed by default.  The number of kills / values (i.e. "x 135") currently doesn't display anything.  So even though it's disabled, I think it's reasonable to add that as a hover item.
2. Both figures are valuable to know - and I think there'd be interest from many in seeing both.  The hover text is simply the one you don't normally want to see.